base/setup/usetup/bootsup.c

```diff
@@ -2552,7 +2579,7 @@ InstallFatBootcodeToPartition(
 
 static
 NTSTATUS
-InstallExt2BootcodeToPartition(
+InstallBtrfsBootcodeToPartition(
```

Comment:
I have this point. If you don't agree with it, I can change :)

Our current setup implementation doesn't distinguish "Linux" filesystems between each other, and @HBelusca is going to enable new FS detection and installation code some time :)
But now usetup can support only one "Linux" FS and I decided to just change the names because code is almost the same for ext4 or btrfs.
If somebody will implement ext4 support too, he/she can just copy btrfs code and change names once more)

Reply:
Let's duplicate the code, keep the ext2 code and surround it with an #ifdef 0 and  add a comment thay for now the only linux partition type recognised by the setup is btrfs.

Reply:
Taking over the Ext2 is not a big deal. If you remember Peter Hater's patches, this code is broken and he was fixing it in order to get proper setup with ext2. So just dropping it isn't a big issue to me.